### DISCUSSION TITLE
Features you'd like to see?

### DISCUSSION THREAD

**Discussion by traverseda** [STEMinds/STEMinds-Docs#9] [Ideas]
I'm looking to put together a "default" firmware that's a bit enhanced from the original. Any thought on what kinds of features people would like to see? Right now the main feature in my fork is that the watering accepts a number of seconds instead of using a simple on or off. I think this is a bit safer.

**Comment by Zontex** [comment]
Hey there!
I think something I've been considering is to add a supported firmware for home-assistant, do you think it would be easy to pull off?I don't have experience with Hassio but I might need to run installation to see how to make it happen.

**Reply by traverseda** [reply]
Probably, I haven't worked with home-assistant in the past (I normally use straight mqtt with some node-red thrown in) but I think it's probably like a yaml file that says what the device exposes? I've worked with zigbee2mqtt so even if we need to tweak some values to closer match how zigbee2mqtt would expose them I think we can make that work.

**Reply by traverseda** [reply]
A note to my future self: https://www.home-assistant.io/docs/mqtt/discovery/
